torch/_higher_order_ops/flex_attention.py

```diff
@@ -187,6 +187,11 @@ def math_attention(
         score_mod: The score_mod function
         other_buffers: Other buffers that are passed to the score_mod function
     """
+    # broadcast query & key along batch dim for paged attention
```

Comment:
Remove the comments that these are for paged attention.

Reply:
Also, we don't want to support interleaving, just broadcasting.

So, basically, we should only support it if `key == 1` and `query > 1`.

Reply:
So, we should just call `expand`, not `repeat_interleave`.